messages/teleporter/message_manager.go

```diff
@@ -237,19 +180,31 @@ func (m *messageManager) SendMessage(signedMessage *warp.Message, parsedVmPayloa
 		return err
 	}
 
+	// Get the correct destination client from the global map
+	destinationClient, ok := m.destinationClients[destinationBlockchainID]
+	if !ok {
+		return fmt.Errorf("relayer not configured to deliver to destination. DestinationBlockchainID=%s", destinationBlockchainID)
+	}
+
+	teleporterMessenger := m.getTeleporterMessenger(destinationClient)
+	teleporterMessageID, err := m.calculateMessageID(teleporterMessenger, signedMessage.SourceChainID, destinationBlockchainID, teleporterMessage.MessageNonce)
```

Comment:
Making an API call here as well to get the message ID just for the sake of logging feels sub-optimal.

Reply:
agreed, will defer for https://github.com/ava-labs/awm-relayer/issues/136